doc/source/advancedexamples.rst

```diff
@@ -445,6 +466,51 @@ be written using :class:`qibo.base.gates.VariationalLayer` as follows:
         circuit.add(gates.CZ(0, nqubits - 1))
     circuit.add((gates.RY(i, theta) for i in range(nqubits)))
 
+.. _vqc-example:
+
+How to write a custom VQC optimization?
```

Comment:
I may be wrong in this but I was a bit confused about the VQC acronym for a few moments, so I would change this to
```suggestion
How to write a custom variational circuit optimization?
```
because I think the VQC acronym is not as commonly used as the VQE.

In the text we could write explicitly "Variational Quantum Circuit (VQC)" for the first time and just use VQC after that.